app/assets/javascripts/discourse/tests/acceptance/sidebar-plugin-api-test.js

```diff
@@ -0,0 +1,255 @@
+                {
+                  id: "browseChannels",
+                  title: "Browse channels",
+                  action: () => {},
+                },
+                {
+                  id: "settings",
+                  title: "Settings",
+                  action: () => {},
+                },
+              ];
+            }
+            get links() {
+              return [
+                new (class extends BaseCustomSidebarSectionLink {
+                  get name() {
+                    "random-channel";
+                  }
+                  get route() {
+                    return "discovery.latest";
+                  }
+                  get model() {
+                    return false;
+                  }
+                  get title() {
+                    return "random channel title";
+                  }
+                  get text() {
+                    return "random channel text";
+                  }
+                  get prefixIcon() {
+                    return "hashtag";
+                  }
+                  get prefixIconColor() {
+                    return "FF0000";
+                  }
+                  get prefixIconBadge() {
+                    return "lock";
+                  }
+                  get suffixIcon() {
+                    return "circle";
+                  }
+                  get suffixCssClass() {
+                    return "unread";
+                  }
+                })(),
+                new (class extends BaseCustomSidebarSectionLink {
+                  get name() {
+                    "dev-channel";
+                  }
+                  get route() {
+                    return "discovery.latest";
+                  }
+                  get model() {
+                    return false;
+                  }
+                  get title() {
+                    return "dev channel title";
+                  }
+                  get text() {
+                    return "dev channel text";
+                  }
+                  get prefixIconColor() {
+                    return "alert";
+                  }
+                  get prefixIcon() {
+                    return "hashtag";
+                  }
+                })(),
+              ];
+            }
+          };
+        }
+      );
+    });
+
+    await visit("/");
+    assert.strictEqual(
+      query(".sidebar-section-chat-channels .sidebar-section-header a").title,
+      "chat channels title",
+      "displays header with correct title attribute"
+    );
+    assert.strictEqual(
+      query(
+        ".sidebar-section-chat-channels .sidebar-section-header a"
+      ).textContent.trim(),
+      "chat channels text",
+      "displays header with correct text"
+    );
+    await click(".edit-channels-dropdown summary");
+    assert.strictEqual(
+      queryAll(".edit-channels-dropdown .select-kit-collection li").length,
+      2,
+      "displays two actions"
+    );
+    const $actions = queryAll(
```

Comment:
```suggestion
    const actions = queryAll(
```

`$` is a jQuery identifier so we shouldn't be using it here since it isn't jQuery related.